ILTransform/ILRewriter.cs

```diff
@@ -237,7 +237,12 @@ private void RewriteFile(string source)
                 {
                     if (isILTest)
                     {
-                        string classLine = $".class public auto ansi Test_{Path.GetFileNameWithoutExtension(source)} extends [mscorlib] System.Object {{";
+                        string testClass = Path.GetFileNameWithoutExtension(source);
+                        if (!Char.IsLetter(testClass[0]))
+                        {
+                            testClass = "Test_" + testClass;
+                        }
```

Comment:
How about `SanitizeIdentifier` instead?  It might be overkill (it's possible that an assembly name may be one of those special tokens), but it already handles this leading non-character case and also handles non-identifier special characters within the name.

Reply:
Should we always use SanitizeIdentifier or just when it does not start with a letter? SanitizeIdentifier is not adding _ when it starts with a special character.

Reply:
Good catch - SanitizeIdentifier needs C# vs IL modes - I'll fix that